scripts/dev.sh

```diff
@@ -131,7 +131,7 @@ else
 fi
 
 tmux new-window -n 'fce' "export ROS_NAMESPACE=$NAME; cd $OCBHAVE/face_track; ./main.py; $SHELL"
-
+tmux new-window -n 'sound' "export ROS_NAMESPACE=$NAME; cd $OCBHAVE/sound; ./audioclass.py; $SHELL"
```

Comment:
yes it is not checked in yet...........and ok I will move the package into
the sound directory

On Fri, Sep 2, 2016 at 9:05 AM, Wenwei <notifications@github.com> wrote:

> In scripts/dev.sh
> <https://github.com/hansonrobotics/HEAD/pull/186#discussion_r77297371>:
>
> > @@ -131,7 +131,7 @@ else
> >  fi
> >
> >  tmux new-window -n 'fce' "export ROS_NAMESPACE=$NAME; cd $OCBHAVE/face_track; ./main.py; $SHELL"
> > -
> > +tmux new-window -n 'sound' "export ROS_NAMESPACE=$NAME; cd $OCBHAVE/sound; ./audioclass.py; $SHELL"
>
> The package should moved to audio directory, not vision.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/hansonrobotics/HEAD/pull/186/files/9bf1e1cd89036faa8ae8b7208924b21c900c5f3c#r77297371>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMm1tSEX1LJBwX4aRcV55AwAfBpgiFbsks5ql7ydgaJpZM4JvhlN>
> .
>